python/cuml/test/dask/conftest.py

```diff
@@ -10,9 +10,11 @@
 
 
 @pytest.fixture(scope="module")
-def cluster():
-
-    cluster = LocalCUDACluster(protocol="tcp", scheduler_port=0)
+def cluster(pytestconfig):
+    args = {'protocol': 'tcp', 'scheduler_port': 0}
+    if pytestconfig.getoption('--use-rmm-pool'):
+        args["rmm_pool_size"] = "2GB"
```

Comment:
Is 2GB pool going to be enough for cuML unit tests?

Reply:
I think it is growable unless max pool size is set. Actually, that suggests we should probably make it smaller and allow it to grow when needed, since we're less concerned about perf. I'm basing this theory on the call to reinitialize here: https://github.com/rapidsai/dask-cuda/blob/a23ea45bafee3176952d72e5a554f3b7e5232b00/dask_cuda/utils.py#L42